docs/intro/tutorial.rst

```diff
@@ -217,7 +216,7 @@ using the :ref:`Scrapy shell <topics-shell>`. Run::
 
 .. note::
 
-   Remember to always enclose urls in quotes when running Scrapy shell from
+   Remember to always enclose urls in quotes when running Scrapy shell from the
    command-line, otherwise urls containing arguments (i.e. ``&`` character)
    will not work.
```

Comment:
```suggestion
   Remember to always enclose URLs in quotes when running Scrapy shell from the
   command line, otherwise URLs containing arguments (i.e. ``&`` character)
   will not work.
```